common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -1148,6 +1148,9 @@ def _create_new_cohort_linked_to_content_group(self, new_cohort, cohort_group):
         """
         self.cohort_management_page.add_cohort(new_cohort, content_group=cohort_group)
         # After adding the cohort, it should automatically be selected
+        print new_cohort
+        print self.cohort_management_page.get_selected_cohort()
+        from nose.tools import set_trace; set_trace()
```

Comment:
@clrux I am assuming these are for debugging, I am placing a comment just to make sure that these are not forgotten :)